src/container.js

```diff
@@ -63,9 +63,9 @@ function injectProperties(container, fn, instance) {
     for (let property in dependencies) {
       instance[property] = container.get(dependencies[property]);
     }
-  }
-  if (instance.postConstruct !== undefined) {
-    instance.postConstruct();
+    if (instance.postConstruct !== undefined) {
```

Comment:
Ok. After thinking some more, I would really like to rename this to "afterConstructor". The "after" naming scheme is used elsewhere in Aurelia so this would help to make that consistent. I'm still a little unsure about adding this though. However, changing the name would make me feel a little better about it.